### ISSUE TITLE
[Feature Request] Easy tags section reading (custom sort ordering)

### ISSUE THREAD

**Issue by zFerry98** [hkalexling/Mango#183]
**Is your feature request related to a problem? Please describe.**
Searching for specific tags becomes difficult when there are so many tags.
![image](https://user-images.githubusercontent.com/17975351/112877547-3068d380-90c7-11eb-8831-b5c60dcf20bc.png)
I was looking for the "nudity" tag to understand how much they need to be added to my personal list to warn that they are manga with mature content ... even now I have not found the tag 😂 (ok, from a pc you do CTRL + F but on android it becomes uncomfortable if you use fullscreen mode).

Other problem:
Being able to use two or more tags at the same time for searching.

**Describe the solution you'd like**
A system to reorder the view by:
Name, Number of repetitions (and if feasible, date of addition).
Select multiple tags and view only the series that include them.

**Describe a small use-case for this feature request**
Do not become fools for 5 minutes to search for a tag or understand if that series includes other desired tags.

**Additional context**
I love the black theme... seeing those white rectangles is a strong punch in the eye 🙃 (I know! I'm a pain in the ass).

**Comment by Leeingnyo** [comment]
It would be nice that there are a filter and sort options for tags!

Searching with multiple tags may be another issue and could be an advanced search function, which might be another big plan, needed design

colors... umm... I'll try some alternatives

**Comment by hkalexling** [comment]
@Leeingnyo Yes I think filtering and sorting on the tags page should be fairly simple to implement, and it should cover most use-cases. For multiple tags, I think we can use the same UI as when adding tags to a title.
